src/app/test/test.component.ts

```diff
@@ -12,18 +12,24 @@ import { DeleteModalComponent } from './delete-modal/delete-modal.component';
 import { ToastService } from '../shared/services/toast.service';
 import { TabService } from '../shared/services/tab.service';
 import { UpdatePathSettings } from '../shared/interfaces/update-path-settings';
+import { View } from '../shared/interfaces/view';
+import { TestListItem } from '../shared/interfaces/test-list-item';
+import { CurrentTestView } from '../shared/interfaces/current-test-view';
+import { CompareReport } from '../shared/interfaces/compare-report';
+import { OptionsSettings } from '../shared/interfaces/options-settings';
+import { TargetWithFiles } from '../shared/interfaces/target-with-files';
 
 @Component({
   selector: 'app-test',
   templateUrl: './test.component.html',
   styleUrls: ['./test.component.css'],
 })
 export class TestComponent implements OnInit {
-  reports: any[] = [];
+  reports: TestListItem[] = [];
   reranReports: ReranReport[] = [];
   generatorStatus: string = 'Disabled';
```

Comment:
Could you make this a boolean, seeing as this can only be enabled or disabled. If that string value is also shown in the HTML, please make a pipe for it.

Reply:
I could definitely make this a boolean. It's only used in 1 html file (test.component.html), Would a pipe be better than something like making a function that returns the disabled/enabled string or using a ternary in the html?

Reply:
This has been implemented